discovery-provider/src/queries/get_premium_track_signatures.py

```diff
@@ -348,8 +361,12 @@ def _does_user_own_sol_nft_collection(
             )
             metadata_accounts = list(map(_get_metadata_account, nft_mints))
             datas = asyncio.run(_decode_metadata_accounts_async(metadata_accounts))
+            datas = list(filter(lambda data: data, datas))
```

Comment:
is this just checking for nulls? I feel like it might be nice to be explicit - `data is not None`

Reply:
fair! will change